synthesizer/src/vm/finalize.rs

```diff
@@ -1641,11 +1696,6 @@ finalize compute:
         // Check that the storage was not updated.
         let program_id = ProgramID::from_str("testing.aleo").unwrap();
         let mapping_name = Identifier::from_str("entries").unwrap();
-        let value = vm
-            .finalize_store()
-            .get_value_speculative(program_id, mapping_name, &Plaintext::from(Literal::Address(address)))
-            .unwrap();
-        println!("{:?}", value);
```

Comment:
Why was this removed?

Reply:
It appears to be a dangling `println` statement. 
`value` does not appear to be use elsewhere.